src/pages/UpdateServers.js

```diff
@@ -983,6 +984,22 @@ class UpdateServers extends BaseUpdateWizardPage {
     const autoServers = this.state.servers.filter(s => s.source !== 'manual');
     const manualServers = this.state.servers.filter(s => s.source === 'manual');
 
+    let serverStates = {};
+
+    // servers that are not compute don't have a active/deactive state
+    this.props.model?.get('inputModel')?.get('servers')?.toJS()?.forEach(s => {
+      if(!s.role.includes('COMPUTE')) {
```

Comment:
is this how we check for compute roles in other spots? I dont remember what we do on the "add compute" page :-/  @guglo would probably know

Reply:
we do exactly this on line 262 of this file.

Reply:
we have a util used in the file called isComputeNode, it would be better to use if(!isComputeNode(s)) here. 
at line 262 .it can be updated to filter(s => isComputeNode(s))